db/schema.rb

```diff
@@ -196,10 +196,11 @@
     t.datetime "created_at", null: false
     t.uuid "form_id", null: false
     t.boolean "is_current", default: true, null: false
-    t.integer "sequence", default: 1, null: false
+    t.string "number"
```

Comment:
This should be null false and have a reasonable limit. You'll have to add the null false constraint after you populate.